### PR TITLE
edit device

### PR DESCRIPTION
**Pull request by t-mon**

### CODE REVIEW

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/228/
Test FAILed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/229/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/230/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/231/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/232/
Test PASSed.

File: libguh/devicemanager.cpp [hunk 1]
```diff
@@ -331,6 +355,106 @@ DeviceManager::DeviceError DeviceManager::addConfiguredDevice(const DeviceClassI
     return addConfiguredDeviceInternal(deviceClassId, descriptor.params(), deviceId);
 }
 
+
+/*! Edit the \l{ParamList}{Params} of a configured device with the given \a deviceId to the new given \a params.
+ *  The given parameter \a fromDiscovery specifies if the new \a params came
+ *  from a discovery or if the user set them. If it came from discovery not editable will be changed too.
+ *
+ *  Returns \l{DeviceError} to inform about the result. */
+DeviceManager::DeviceError DeviceManager::editDevice(const DeviceId &deviceId, const ParamList &params, const bool &fromDiscovery)
```
Comment (mzanetti) [inline libguh/devicemanager.cpp]: const bool &fromDiscovery. const & not needed on bools I'd say. for consistency I'd drop it.

File: plugins/generateplugininfo.py [hunk 2]
```diff
@@ -8,100 +8,105 @@
...
+	outputfile.write("%s\n" % line)
 
 def extractVendors(pluginMap):
-    for vendor in pluginMap['vendors']:
-        try:
-	    out("VendorId %sVendorId = VendorId(\"%s\");" % pluginMap["idName"], pluginMap["id"])
-        except:
-            pass
-        extractDeviceClasses(vendor)
+	for vendor in pluginMap['vendors']:
```
Comment (mzanetti) [inline plugins/generateplugininfo.py]: tabs indentation!

**Comment by t-mon** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/234/
Test FAILed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/235/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/236/
Test PASSed.

**Comment by t-mon** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://guh.guru:8080/job/guh-pull-request-builder/237/
Test PASSed.

**Pull request merged**